src/appleseed/foundation/image/genericimagefilewriter.h

```diff
@@ -5,8 +5,7 @@
 //
 // This software is released under the MIT license.
 //
-// Copyright (c) 2010-2013 Francois Beaune, Jupiter Jazz Limited
-// Copyright (c) 2014-2018 Francois Beaune, The appleseedhq Organization
+// Copyright (c) 2018 Thomas Manceau, The appleseedhq Organization
```

Comment:
I understand that most of the code in this file has changed, but you should retain the copyright notice anyway.

Reply:
Done